build.js

```diff
@@ -12,7 +12,9 @@ const plugins = [
   require("autoprefixer"),
   require("postcss-import"),
   require("postcss-nested"),
-  require("postcss-css-variables"),
+  require("postcss-css-variables")({
+    preserve: true
+  }),
```

Comment:
```suggestion
```

Let's remove this altogether

Reply:
Hi @khang-nd. I'll update my PR when I get a chance, cheers. :)

Reply:
Hey @khang-nd, after removing the `postcss-css-variables` package, I've noted that some of the UI becomes broken. Please see the below examples:

### Title bar padding
![image](https://github.com/khang-nd/7.css/assets/68851569/15541690-6a64-4b7e-9927-8bca952a1fb3)

### Balloon / tooltip handles
![image](https://github.com/khang-nd/7.css/assets/68851569/a36e9b29-da90-4e34-8062-158930548f77)

### Radio button positioning
![image](https://github.com/khang-nd/7.css/assets/68851569/35df5f78-d0b7-45dd-947e-154876fc2ac3)

### Range sliders
Track disappears and handle position is offset.
![image](https://github.com/khang-nd/7.css/assets/68851569/2dd3c968-d678-4300-a433-210467e9182b)
![image](https://github.com/khang-nd/7.css/assets/68851569/caf09bfb-bd0d-48a8-9494-5f0e78d29227)

### Status bar
Displays a wider frame, where it should just display a 1 pixel line.
![image](https://github.com/khang-nd/7.css/assets/68851569/57687a46-f51a-46d4-b8b5-98222e360ef8)

Those are all the issues I could see at a quick glance, I'm happy to try and address them as a part of this PR unless you think it warrants a new PR or a renaming of this PR to be more broad?

### Steps to recreate
Run `npm uninstall postcss-css-variables` and then `npm run build`.

Keen to hear your thoughts, cheers.

Reply:
@liam-mills I'm more than happy if you could address those issues within this PR, but it's totally fine if you don't want to, I can tackle them later 🙂

Reply:
@khang-nd I'll have a look at fixing these up when I get the time.

Reply:
Hey @khang-nd, new commit should resolve all of the above issues I found.

Resolving the slider styling was a bit of headache until I remembered something about vendor specific pseudo selectors not working if they are chained together. Separating the selectors resolved this.

Let me know if you come across any other styling inconsistencies and I'll look at them as well, but hopefully we should be all good.

Cheers.